website/docs/terraform/environment-variables.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Logging Env Variables
```

Comment:
We should be linking to this page from the appropriate context in the overview page.

Reply:
It is listed automatically on the side bar. don't worry.

Reply:
Yeah, I mean like linking from it with a description along the lines of "for further configuration options refer to this page". This gives it a little more visibility.

Reply:
Unfortunately, what you're saying is not fit nicely with the content of the overview page. Let me think about it next time as I'm also planning to re-org this doc somehow in the near future.

Reply:
Sure thing. We can definitely iterate. This PR is a great addition!! ✨